java/ql/src/semmle/code/java/security/UnsafeCertTrust.qll

```diff
@@ -0,0 +1,144 @@
+/** Provides classes and predicates to reason about unsafe certificate trust vulnerablities. */
+
+import java
+private import semmle.code.java.frameworks.Networking
+private import semmle.code.java.security.Encryption
+private import semmle.code.java.dataflow.DataFlow
+private import semmle.code.java.dataflow.DataFlow2
+
+/**
+ * The creation of an object that prepares an SSL connection.
+ *
+ * This is a source for `SslEndpointIdentificationFlowConfig`.
```

Comment:
I took inspiration from other libraries with similar comments (e.g. https://github.com/github/codeql/blob/9362ae0687857cdb2e791d9a9558e99ffa96c54d/java/ql/src/semmle/code/java/security/LdapInjection.qll#L20)

Arguably, this class won't be used anywhere else since it's specifically designed as a source for that taint tracking configuration, hence the comment. In the event that it actually gets reused for something else, the comment could be removed.